engines/grim/gfx_tinygl.cpp

```diff
@@ -1044,7 +1044,14 @@ void GfxTinyGL::drawBitmap(const Bitmap *bitmap, int x, int y, uint32 layer) {
 	Graphics::BlitImage **b = (Graphics::BlitImage **)bitmap->getTexIds();
 
 	if (bitmap->getFormat() == 1) {
+		if (bitmap->getHasTransparency()) {
```

Comment:
You mean on the `if` itself ? I copied it from `gfx_opengl.cpp` (without completely sync'ing the method either...).